test/ServiceManagerTest.php

```diff
@@ -20,16 +21,23 @@
 use function get_class;
 
 /**
+ * @see ConfigInterface
```

Comment:
I really like how you used `@see` to import types that are otherwise elided by the CS tooling :D